docs: tw798 (#3270)
OffchainLabs/arbitrum-docs: docs/for-devs/third-party-docs/Reown/reown.mdx

## AppKit

AppKit is a powerful, free, and fully open-source SDK for developers looking to integrate wallet connections and other Web3 functionalities into their apps on any EVM and non-EVM chain. In just a few simple steps, you can provide your users with wallet access, one-click authentication, social logins, and notifications—streamlining their experience while enabling advanced features like on-ramp functionality, in-app token swaps and smart accounts.

> ## How to get started with AppKit on Arbitrum?

Reviewer: ```suggestion
## How to get started with AppKit on Arbitrum
```

And that’s it! You have now learned how to create a simple app using AppKit that allows users to connect their wallet and interact with the Arbitrum network.

> **Reown AppKit** is a powerful solution for developers looking to integrate wallet connections and other Web3 functionalities into their apps on any EVM chain. In just a few simple steps, you can provide your users with wallet access, one-click authentication, social logins, and notifications—streamlining their experience while enabling advanced features like on-ramp functionality and smart accounts. By following this guide, you'll quickly get up and running with Reown’s AppKit, enhancing your app’s user experience and interaction with blockchain technology.

Reviewer: ```suggestion
**Reown AppKit** is a powerful solution for developers looking to integrate wallet connections and other Web3 functionalities into their apps on any EVM chain. In just a few simple steps, you can provide your users with wallet access, one-click authentication, social logins, and notifications—streamlining their experience while enabling advanced features like on-ramp functionality and smart accounts. By following this guide, you'll quickly get up and running with Reown's AppKit, enhancing your app's user experience and onchain interactions.
```

> You can view the complete code repository [here](https://github.com/rohit-710/reown-appkit-evm).

Reviewer: ```suggestion
You can view the [complete code repository](https://github.com/rohit-710/reown-appkit-evm).
```
